### PR TITLE
Update UG

### LINKED ISSUE #47: Update commands related to pet tracker in user guide

### PR DESCRIPTION
**Pull request by tianqi77**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -151,18 +151,18 @@ Provides statistics about the pet tracker and schedule system, such as total num
 
 Adds a pet to the pet tracker system.
 
-Format: `addpet n/NAME a/AGE s/SPECIES b/BREED g/GENDER f/FOOD [t/TAG]...`
+Format: `addpet n/NAME g/GENDER b/DATE OF BIRTH s/SPECIES f/FOOD : AMOUNT [t/TAG]...`
```
Comment (Zhu-Ruicong) [inline docs/UserGuide.adoc:154]: is the space before and after : intended? Our current implementation is `food:amount` not `food : amount` right?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -151,18 +151,18 @@ Provides statistics about the pet tracker and schedule system, such as total num
...
 
 ****
-- The age must be a positive integer, e.g. 1, 2, 3, ..., denoting how many years old the pet is.
-- The gender must be either `F` or `M`.
-- Food is specified as a quantity of weekly consumption in an arbitrary unit.
-- A pet can have any number of tags (including 0)
+- The date of birth must be in the format of d/M/YYYY, e.g. 01-01-2019, 1-7-2018
```
Comment (Zhu-Ruicong) [inline docs/UserGuide.adoc]: DOB format

File: docs/UserGuide.adoc (hunk 1 above)
Comment (tianqi77) [reply docs/UserGuide.adoc:154]: actually it's the same, cuz the spaces will be trimmed

File: docs/UserGuide.adoc (hunk 2 above)
Comment (tianqi77) [reply docs/UserGuide.adoc]: ok I go fix later. thx:D

**Pull request merged**